internal/js/console_test.go

```diff
@@ -378,7 +378,6 @@ func TestFileConsole(t *testing.T) {
 							f, err := os.CreateTemp(t.TempDir(), "") //nolint:forbidigo // fix with https://github.com/grafana/k6/issues/2565
 							require.NoError(t, err)
 							logFilename := f.Name()
-							defer os.Remove(logFilename) //nolint:errcheck,forbidigo // fix with https://github.com/grafana/k6/issues/2565
```

Comment:
I think this line is no longer required, as `t.TempDir()` is responsible, thanks to `t.Cleanup()` behavior, to completely remove the temporary directory created for this test, once it finishes. Right?

Reply:
Seems reasonable - I did remove some lines like this from some places, but likely missed some as well